docs/core/testing/unit-testing-mstest-sdk.md

```diff
@@ -36,75 +39,108 @@ When you `build` the project all the needed components will be restored and inst
+By default, MSTest SDK will rely on MSTest runner but you can easily switch to VSTest by adding the following property `<UseVSTest>true</UseVSTest>`.
 
-For example, to enable the crash dump extension in the [Microsoft.Testing.Extensions.CrashDump](https://www.nuget.org/packages/Microsoft.Testing.Extensions.CrashDump) NuGet package use the following property `EnableMicrosoftTestingExtensionsCrashDump`:  
+## Extending MSTest runner
 
-  ```xml
-  <Project Sdk="MSTest.Sdk/3.3.0">
-    <PropertyGroup> 
-        <TargetFramework>net8.0</TargetFramework>
-        <EnableMicrosoftTestingExtensionsCrashDump>true</EnableMicrosoftTestingExtensionsCrashDump>
-    </PropertyGroup>
-    <!-- references to the code to test -->
-  </Project>
-  ```
+You can customize `MSTest runner` experience through a set of [NuGet package extensions](./unit-testing-platform-extensions.md). To simplify and improve this experience, MSTest SDK introduces two features.
 
-### Available extensions
+### MSTest Runner profile
 
-| Extension package | MSBuild property | 'Default' profile | 'AllMicrosoft' profile |
-| -------- | ----------- | ----------- | ----------- |
-| `Microsoft.Testing.Extensions.CodeCoverage` | `<EnableMicrosoftTestingExtensionsCodeCoverage>true</EnableMicrosoftTestingExtensionsCodeCoverage>` | ✔️ | ✔️ |
-| `Microsoft.Testing.Extensions.TrxReport` | `<EnableMicrosoftTestingExtensionsTrxReport>true</EnableMicrosoftTestingExtensionsTrxReport>` | ✔️ | ✔️ |
-| `Microsoft.Testing.Extensions.CrashDump` | `<EnableMicrosoftTestingExtensionsCrashDump>true</EnableMicrosoftTestingExtensionsCrashDump>` | ❌ | ✔️ |
-| `Microsoft.Testing.Extensions.HangDump` | `<EnableMicrosoftTestingExtensionsHangDump>true</EnableMicrosoftTestingExtensionsHangDump>` | ❌ | ✔️ |
-| `Microsoft.Testing.Extensions.Retry` | `<EnableMicrosoftTestingExtensionsRetry>true</EnableMicrosoftTestingExtensionsRetry>` | ❌ | ✔️ |
-| `Microsoft.Testing.Extensions.HotReload` | `<EnableMicrosoftTestingExtensionsHotReload>true</EnableMicrosoftTestingExtensionsHotReload>` | ❌ | ✔️ |
+The concept of profile, allows you to select the default set of configurations and extensions that will be applied to your test project.
 
-The `MSTest.Sdk` simplifies the registration of extensions by including a pre-packaged set of them under different profiles, as shown in the preceding table.
+You can set the profile using the property `TestingExtensionsProfile` to one of the following 3 profiles:
 
-To enable a specific profile, use the `TestingExtensionsProfile` MSBuild property.  
-For example, if you wish to enable all Microsoft testing extensions, you can configure the `TestingExtensionsProfile` as shown below:
+- `None` - no extension is added,
+- `Default` - the default profile when none is explicitly set. Depending on the version of the SDK, the recommended list of extensions.
```

Comment:
"none is explicitly set". This is a little bit confusing with `None`. I will change the wording to something like `when this property is not provided/missing`